src/bin/nydusd/api_server_glue.rs

```diff
@@ -443,5 +443,8 @@ impl ApiServerController {
                 );
             }
         }
+        if let Some(apisock) = self.sock.as_ref() {
+            std::fs::remove_file(apisock).unwrap_or_default();
```

Comment:
Can we give a warning if any error happens?

Reply:
ok, resolved.